spec/rails_helper.rb

```diff
@@ -14,7 +14,7 @@
 require 'timecop'
 
 # load all files in support folders
-Dir[TestTrackRailsClient::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
+Dir[TestTrackRailsClient::Engine.root.join("spec/support/**/*.rb")].sort.each { |f| require f }
```

Comment:
`Dir`'s default sorting is OS/filesystem-dependent, so this is a pretty smart rule.